projects/packages/my-jetpack/_inc/components/product-detail-card/index.jsx

```diff
@@ -210,6 +210,19 @@ const ProductDetailCard = ( { slug, onClick, trackButtonClick, className } ) =>
 						<Text>{ __( 'Active on your site', 'jetpack-my-jetpack' ) }</Text>
 					</div>
 				) }
+
+				{ 'search' === slug && (
```

Comment:
I'm not 100% sure about handling this particular situation in this way. Maybe we could add a new `note` field, or whatever the name is, to be rendered in this part of the product component.
Anyway, we can iterate later.

{ __(
							"Pricing will automatically adjust based on the number of records in your search index. If you grow into a new pricing tier, we'll let you know before your next billing cycle.",
							'jetpack-my-jetpack'
						) }

Reply:
Yes, let's avoid this

Reply:
Happy to defer to whatever approach would be canonical for My Jetpack here.

Reply:
I think the first thing that we need to define properly is the purpose of this section from the UI perspective:
<img width="936" alt="image" src="https://user-images.githubusercontent.com/77539/168469745-5a16bf9a-bfb1-408b-adcc-6982927dcd57.png">

@ederrengifo @keoshi, I don't see this section in the Figma RNA designs. Do you think we can put a name and define a purpose for it? Nothing too complex. I'd say something like `info`, `note`, ... but surely you can raise up better ideas here.

Reply:
We've allocated a space for this type of context which you can see in the [RNA: Dialog Screen page](https://www.figma.com/file/dRt1pX85T68o5UNYK6AZro/?node-id=924%3A2553)

<img width="819" alt="image" src="https://user-images.githubusercontent.com/390760/168606107-16b333c4-7d8d-437e-9fec-65f423d2619c.png">

As for a name, I'd call it `Supporting information` since that covers many of our use cases: ToS, pricing info, help links, etc.

Reply:
Thanks, Filipe. Exactly what we need!

Reply:
Resolved in 95ce3d1 🙆‍♂️